dashboard/config/scripts/levels/New PoemBot Project.level

```diff
@@ -0,0 +1,103 @@
+    "start_animations": "{\r\n  \"orderedKeys\": [\r\n    \"5a4258fa-c1a8-4898-a44e-c7d7419ebfa4\",\r\n    \"5c292f95-929d-48f3-9432-0f00a8c77934\",\r\n    \"a54f4f7d-6764-4345-abaa-b651abdca33e\",\r\n    \"64c242da-16c3-4261-9c57-936ce1f90aad\"\r\n  ],\r\n  \"propsByKey\": {\r\n    \"5a4258fa-c1a8-4898-a44e-c7d7419ebfa4\": {\r\n      \"name\": \"healthy\",\r\n      \"sourceUrl\": \"https://levelbuilder-studio.code.org/media?u=https%3A%2F%2Flevelbuilder-studio.code.org%2Fmedia%3Fu%3Dhttps%253A%252F%252Fstudio.code.org%252Fapi%252Fv1%252Fanimation-library%252Fspritelab%252FCdlNOvtih6CV7QUekCCPH7e11tMwCGQh%252Fcategory_characters%252Fpurple_monster.png\",\r\n      \"frameSize\": {\r\n        \"x\": 280,\r\n        \"y\": 290\r\n      },\r\n      \"frameCount\": 1,\r\n      \"looping\": true,\r\n      \"frameDelay\": 2,\r\n      \"version\": \"CdlNOvtih6CV7QUekCCPH7e11tMwCGQh\",\r\n      \"categories\": [\r\n        \"characters\"\r\n      ]\r\n    },\r\n    \"5c292f95-929d-48f3-9432-0f00a8c77934\": {\r\n      \"name\": \"sick\",\r\n      \"sourceUrl\": \"https://levelbuilder-studio.code.org/media?u=https%3A%2F%2Flevelbuilder-studio.code.org%2Fmedia%3Fu%3Dhttps%253A%252F%252Fstudio.code.org%252Fmedia%253Fu%253Dhttps%25253A%25252F%25252Flevelbuilder-studio.code.org%25252Fv3%25252Fanimations%25252FVMkrxJPPGixIsTbGRHNrmw%25252F5c292f95-929d-48f3-9432-0f00a8c77934.png%25253Fversion%25253D7pTy7xkAeI5qrU9scI3nWmKb7CzBMlfU\",\r\n      \"frameSize\": {\r\n        \"x\": 280,\r\n        \"y\": 290\r\n      },\r\n      \"frameCount\": 1,\r\n      \"looping\": true,\r\n      \"frameDelay\": 12,\r\n      \"version\": \"7pTy7xkAeI5qrU9scI3nWmKb7CzBMlfU\"\r\n    },\r\n    \"a54f4f7d-6764-4345-abaa-b651abdca33e\": {\r\n      \"name\": \"healthy_mask\",\r\n      \"sourceUrl\": \"https://levelbuilder-studio.code.org/media?u=https%3A%2F%2Flevelbuilder-studio.code.org%2Fmedia%3Fu%3Dhttps%253A%252F%252Fstudio.code.org%252Fv3%252Fanimations%252F1alB-zQ8FPOdcMCQ8JPl2RP80uKiLDna7JAYAYRze7A%252Fa54f4f7d-6764-4345-abaa-b651abdca33e.png%253Fversion%253D6x.8TVua7c1V7jHF1sg5T3.LfPLdT7PU\",\r\n      \"frameSize\": {\r\n        \"x\": 280,\r\n        \"y\": 290\r\n      },\r\n      \"frameCount\": 1,\r\n      \"looping\": true,\r\n      \"frameDelay\": 12,\r\n      \"version\": \"6x.8TVua7c1V7jHF1sg5T3.LfPLdT7PU\"\r\n    },\r\n    \"64c242da-16c3-4261-9c57-936ce1f90aad\": {\r\n      \"name\": \"sick_mask\",\r\n      \"sourceUrl\": \"https://levelbuilder-studio.code.org/media?u=https%3A%2F%2Flevelbuilder-studio.code.org%2Fmedia%3Fu%3Dhttps%253A%252F%252Fstudio.code.org%252Fv3%252Fanimations%252F1alB-zQ8FPOdcMCQ8JPl2RP80uKiLDna7JAYAYRze7A%252F64c242da-16c3-4261-9c57-936ce1f90aad.png%253Fversion%253DYm9Z9V3PzU8puG_vRfdn2fsPcYtbd4.2\",\r\n      \"frameSize\": {\r\n        \"x\": 280,\r\n        \"y\": 290\r\n      },\r\n      \"frameCount\": 1,\r\n      \"looping\": true,\r\n      \"frameDelay\": 12,\r\n      \"version\": \"Ym9Z9V3PzU8puG_vRfdn2fsPcYtbd4.2\"\r\n    }\r\n  }\r\n}",
```

Comment:
Should this file be trimmed at some point?

Reply:
I cleared out the `start_animations` because I know that's safe, but I don't want mess too much with this file since it's generated from LB